### PR TITLE
Add context manager for colorama

### PR DESCRIPTION
**Pull request by digwanderlust**
This enables you to handle the stdout replacement with a with statement rather than manually keeping track of it... for example:

with colorama_text():
      print Fore.GREEN, 'text is green'
      print Fore.RESET, 'text is back to normal'

print 'text is back to stdout'

### CODE REVIEW

**Comment by digwanderlust** [comment]
Are there other steps I should follow to get this merged?

**Comment by wiggin15** [comment]
This looks good to me (I wasn't sure about the name of the context method but I guess it's alright). This will also resolve #51.

File: colorama/initialise.py [hunk 1]
```diff
@@ -17,7 +18,6 @@
 def reset_all():
     AnsiToWin32(orig_stdout).reset_all()
 
-
```
Comment (wiggin15) [inline colorama/initialise.py]: There are two blank lines between functions in this file, so this line shouldn't be removed

File: colorama/initialise.py [hunk 2]
```diff
@@ -53,6 +53,14 @@ def deinit():
...
 
 
+@contextlib.contextmanager
+def colorama_text(*args):
+    init(*args)
+    try:
+        yield
+    finally:
+        deinit()
+
```
Comment (wiggin15) [inline colorama/initialise.py]: There should be two blank lines following this function too

File: colorama/initialise.py [hunk 3]
```diff
@@ -53,6 +53,14 @@ def deinit():
         sys.stderr = orig_stderr
 
 
+@contextlib.contextmanager
+def colorama_text(*args):
```
Comment (wiggin15) [inline colorama/initialise.py]: Maybe we should add **kwargs too to pass to 'init' so we can use the context manager with the keywords

**Comment by wiggin15** [comment]
Sorry for the new batch of comments. I should have done a better review before. Thanks for fixing.

**Comment by digwanderlust** [comment]
No worries, just finished up the changes... let me know if you see anything
else.

**Comment by wiggin15** [comment]
Thanks!

**Pull request merged**